lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,37 @@
+local CATEGORY_NAME = "Fun"
+
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+    for _, v in pairs( targets ) do
```

Comment:
instead of v its better to be more descriptive and call it ply `for _, ply in pairs( targets ) do`